julia/ex01_single_cavity.ipynb

```diff
@@ -366,14 +382,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  [info] Steady-state obtained in   0.097629 seconds (217.32 k allocations: 22.742 MiB, 89.94% compilation time)\n"
+     ]
+    }
+   ],
    "source": [
     "@time begin\n",
     "# Diagonalize\n",
-    "_, pss = eigs(L ,nev=1,sigma=0);\n",
-    "_,  Iv = eigs(L',nev=1,sigma=0);\n",
+    "_, vecs_r = eigsolve(L ,k=1,sigma=0);\n",
```

Comment:
Any advantage in replacing `eigs` with `eigsolver`, apart from getting people used to `QuantumToolbox`? It actually looks a bit more verbose than before.

Reply:
Arpack is very good, but it doesn't support multithreading. So, at the first case when the students need to sweep a parameter, they would face a crash using Arpack. In my opinion it is better to show them a more stable function.

Reply:
It is more verbose just because it always returns a vector of states, which however I find it easier to understand. In the previous case, the return type was already the state, but then if you want to get multiple states, you have to divide the matrix in columns.

Reply:
Ok, sounds good!